apps/gauzy/src/app/pages/dashboard/employee-charts/employee-charts.component.html

```
@@ -5,21 +5,21 @@
     class="select-charts"
-    filled
-    status="success"
+    outline
+    status="basic"
     [(selected)]="selectedChart"
-    size="tiny"
+    size="small"
     >
     <nb-option [value]="EmployeeChartEnum.BAR">
-      <img width="auto" height="24" style="margin-right: 10px" src="assets/images/others/bar-chart.svg" />
+      <img width="auto" height="16" style="margin-right: 8px" src="assets/images/others/bar-chart.svg" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)